lib/charms/opensearch/v0/helper_charm.py

```diff
@@ -203,3 +203,14 @@ def mask_sensitive_information(cmd: str) -> str:
     pattern = re.compile(r"(-tspass\s+|-kspass\s+|-storepass\s+|-new\s+|pass:)(\S+)")
 
     return re.sub(pattern, r"\1" + "xxx", cmd)
+
+
+def data_role_in_cluster_fleet_apps(charm: "OpenSearchBaseCharm") -> bool:
```

Comment:
Could you move this one to `helper_cluster.ClusterTopology`?

Reply:
done.